scripts/core/editor3/components/Editor3Component.tsx

```diff
@@ -212,7 +212,7 @@ export class Editor3Component extends React.Component<IProps> {
         return !canDropMedia(e, this.props);
     }
 
-    handleDropOnEditor(selection: SelectionState, dataTransfer: Object, isInternal: DraftDragType): DraftHandleValue {
+    handleDropOnEditor(selection: SelectionState, dataTransfer: any, isInternal: DraftDragType): DraftHandleValue {
```

Comment:
`dataTransfer` argument is not used at all. Can you remove it?

Reply:
@tomaskikutis the argument is coming from https://draftjs.org/docs/api-reference-editor.html#handledrop